File: C_ar05.py
chk = [0] * 30  
  
n = int(input())  
val = 0  
arr = list(map(int, input().split()))  
for i in range(n):  
    a, b = arr[i*2:i*2+2]  
    for i in range(a, b):  
        chk[i] += 1  
        val = max(chk[i], val)  
  
print(val)

File: C_AR05.c
int arr[8192]; 
#include <stdio.h> 
int main() { 
    int n; scanf("%d", &n); 
    for (int i = 0; i < n; i++) { 
        int a, b; 
        scanf("%d %d", &a, &b); 
        for (int i = a; i < b; i++) { 
            arr[i]++; 
        } 
    } 
    int u=  0; 
    for (int i = 0; i < 8192; i++) { 
        if (arr[i] > u) u = arr[i]; 
    } 
 
    printf("%d\n", u); 
}  
